optimize/backend/src/test/java/io/camunda/optimize/service/util/DecisionVariableHelperTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.optimize.dto.optimize.query.variable.VariableType;
+import org.junit.jupiter.api.Test;
+
+public class DecisionVariableHelperTest {
+
+  @Test
+  void shouldGetVariableValueFieldReturnCorrectFieldPath() {
+    // given
+    final String variablePath = "testPath";
+
+    // when
+    final String result = DecisionVariableHelper.getVariableValueField(variablePath);
+
+    // then
+    assertThat(result).isEqualTo("testPath.value");
+  }
+
+  @Test
+  void shouldGetVariableMultivalueFieldsReturnUnmodifiableList() {
+    // given
+
+    // when
+    final var result = DecisionVariableHelper.getVariableMultivalueFields();
```

Comment:
We don't use `var` in Optimize, please declare this variable explicitly

Reply:
I think in the mono repo this is now fine and we should adopt the global code standards